site/en/docs/workbox/service-worker-deployment/index.md

```diff
@@ -116,7 +116,7 @@ Browsers set storage quotas, and limits can vary. That's a good reason to be min
 Workbox does a good job of mitigating these issues,
 but storage quotas can still be exceeded.
 You can achieve finer control of caches with the
-[workbox-expiration module](​​/docs/workbox/modules/workbox-expiration).
+[workbox-expiration module](/docs/workbox/modules/workbox-expiration).
```

Comment:
@dartess, this appears to be the same link, and both instances of this URL lead to the proper page—unless I missed something?

Reply:
@malchata Removed some non-printable characters.  The link doesn't work because of them.